homeassistant/components/webostv/media_player.py

```diff
@@ -145,7 +145,7 @@ async def async_signal_handler(self, data):
             }
             await getattr(self, data["method"])(**params)
 
-    async def async_handle_state_update(self):
+    async def async_handle_state_update(self, _client: WebOsClient):
```

Comment:
Note, callback function was changed to include the client:
https://github.com/chros73/bscpylgtv/blob/2487f498c808c5d2193e9ec625e1b031943006f3/bscpylgtv/webos_client.py#L466